_includes/icml/2020/about.icml.html

```diff
@@ -0,0 +1,380 @@
+<!--          <li><del>April 1, 2019: Notification of submission acceptance</del></li>-->
+<!--          <li><del>April 11, 2019: <a -->
+<!--              href="https://travel.state.gov/content/travel/en/us-visas/tourism-visit/visitor.html">-->
+<!--                Visa application</a> deadline</del></li>-->
+<!--          <li>Mach 12, 2019: ICML registration opens at 8am</li>-->
+<!--          <li>June 10, 2019: LXAI Workshop</li>-->
+<!--        </ul>-->
+<!--        <a class="project-link" href="/icml-2019-travel">Travel Grant -->
+<!--            Application</a>-->
+      </div>
+    </div>
+    
+    <!--
+    <div>
+      <h2>Submission Instructions</h2>
+      <p>We strongly encourage students, post-docs and researchers who
+      primarily identify as latinx in all areas of machine learning to submit
+      an abstract describing new, previously, or concurrently published
+      research. We welcome abstract submissions, in theory, methodology, as
+      well as applications. Abstracts may describe completed research or
+      work-in-progress. While the presenting author need not be the first
+      author of the work, we encourage authors to highlight the contribution
+      of latinx individuals — particularly the presenting author — in the
+      abstract.</p>
+      <p>Authors of accepted abstracts will be asked to present their work in
+      a poster session. A few authors will be selected to give 15 minute oral
+      presentations.  Submissions will be peer-reviewed.  Authors are
+      encouraged to sign up to review for LXAI, with a sign-up option
+      available upon submission.</p>
+
+      <h3>Submission Guidelines</h3>
+      <p>Submissions may be up to two pages including all figures and tables,
+      with an additional page for references submitted as a link to a PDF.
+      The submissions should be in a single column. They should be typeset
+      using 11-point or larger fonts and should have at least 1-inch margin
+      all around. Submissions that do not follow these guidelines risk being
+      rejected without consideration of their merits. Submissions must state
+      the research problem, motivation, and technical contribution.
+      Submissions must be self-contained and include all figures, tables, and
+      references.</p>
+      <p>The submission deadline is <strong>11:59 PM (PST), March 22,
+      2019</strong>. Please note that no extensions will be offered for
+      submissions.</p>
+      <a class="project-link" href="/icml-2019-apply">Submit Abstract</a>
+    </div>
+    -->
+
+    <hr>
+    <div id="workshopSponsors" style="color: black">
+      <h2>Sponsors</h2>
+      <p>If you are interested in becoming a sponsor, please check out our
+<!--      <a href="/icml-2020-sponsor">Sponsor Prospectus</a>-->
+        <a href="#workshopSponsors">Sponsor Prospectus</a>
+      and send us message to <strong>lxai-icml-chairs-2020 at accel dot ai</strong>
```

Comment:
Since we are moving to an annual sponsorship model, update the email to the board email at latinxinai@accel.ai.